Server/src/models/AuthenticationModel.ts

```diff
@@ -127,4 +127,33 @@ export class AuthenticationModel {
 
         let connection = getConnection();
     }
+
+
+    //for issue 146
+    static async updateUserPasswordDetail(email: string, passwordHash: string): Promise<boolean> {
+
+        let connection = getConnection();
+        //for password
```

Comment:
If comments are needed we can add them above the method

Reply:
done